cosmicfishpie/LSSsurvey/spectro_obs.py

```diff
@@ -694,7 +695,7 @@ def f_mom(k):
         ff = f_mom(self.k_grid).flatten()
         pp = cosmoF.matpow(zz, self.k_grid).flatten()
         integrand = pp * ff
-        Int = np.trapezoid(integrand, x=self.k_grid)
+        Int = trapezoid(integrand, x=self.k_grid)
```

Comment:
why not integrate.trapezoid as in photo_obs?  Let's unify

Reply:
sure